testStrategies/commonToAllTestStrategies.go

```diff
@@ -448,16 +460,16 @@ func convertStoredValueToRequestFormat(storedValue interface{}, requiredIndex in
 	return requestFormattedValue
 }
 
```

Comment:
GlobalsMap is now truly global (see comment above on line 222 of commonToAllTestStrategies.go).